crates/solver/src/settlement_ranker.rs

```diff
@@ -60,6 +60,12 @@ impl SettlementRanker {
         let name = solver.name();
         match settlements {
             Ok(settlements) => {
+                if settlements.is_empty() {
+                    solver.notify_auction_result(
+                        auction_id,
+                        AuctionResult::Rejected(SolverRejectionReason::NoUserOrders),
+                    );
+                }
```

Comment:
That's not the right rejection reason technically but not sure if it matters.

Reply:
https://github.com/cowprotocol/services/pull/1681/commits/7c2ff985f59f6af80b599ecbc8a1387fdb39d20a